tests/main.c

```diff
@@ -40,10 +39,40 @@
 #include <byte_buf_test.c>
 #include <system_info_tests.c>
 #include <realloc_test.c>
+#include <memory_pool_test.c>
+#include <log_test.h>
 #include <lru_cache_test.c>
 
+/* Enables terminal escape sequences for text coloring. */
+/* https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences */
+#ifdef _MSC_VER
```

Comment:
this is awesome. Could we move this to aws_test_harness and call this function in the run all tests function?

Reply:
Yeah we definitely can :)